classes/class-wc-connect-shipping-method.php

```diff
@@ -360,7 +360,7 @@ public function calculate_shipping( $package = array() ) {
 						'label'     => self::format_rate_title( $rate->title ),
 						'cost'      => $rate->rate,
 						'meta_data' => array(
-							'wc_connect_packages' => json_encode( $rate->packages ),
+							'wc_connect_packages' => WC_Connect_Utils::json_encode_for_db( $rate->packages ),
```

Comment:
Can we avoid all of the JSON headaches here by using PHP's serialization methods? It's not done automatically for rate `meta_data` like it is for post meta, but perhaps it's still better than JSON.